charts/generic-service/values.yaml

```diff
@@ -43,14 +43,16 @@ service:
 
 ingress:
   enabled: false
+  # v1.2 is the new ingress controller, replacing the existing 0.47 ones
+  # instructions on advised upgrade process - https://mojdt.slack.com/archives/CH6D099DF/p1657794137979789
+  # switching this to true with perform a big bang ingress switch with possible downtime as dns changes
+  v1_2_enabled: false
```

Comment:
default is to carry on using old v0.47 controllers

Reply:
Nice way to prevent this from being a breaking change. :) I guess we go up to v2.0.0 if/when we remove this feature switch.

Reply:
one of the issues these days is that some people have automated the upgrade of the charts so don't really want to change their ingresses for them without their knowledge...